src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleFiltering/components/LineageFilter/index.tsx

```diff
@@ -19,31 +19,12 @@ export type LineageFilterType = {
   setSelectedLineages: (lineages: string[]) => void;
 };
 
-// We present a pseudo-option to the user to enable choosing "All" lineages,
-// but internally this means no lineages were chosen to filter down to.
-const ALL_LINEAGES_KEYWORD = "All";
-
 function makeDropdownOption(name: string): DefaultMenuSelectOption {
   return { name: name };
 }
-// Generate only once because we need to reference same object throughout.
-const ALL_LINEAGES_CHOICE = makeDropdownOption(ALL_LINEAGES_KEYWORD);
 
 /**
- * `Dropdown` defaults to checking if option is selected (`value`) by equality.
```

Comment:
I just want to double-check here: we are no longer dynamically generating the the options, right? It's just an array we ingest once and then never have to regenerate and would never accidentally get regenerated as we prep what we display to user? I've definitely been surprised by being bitten by the object-equality aspect instead of value-equality with our dropdown options, so wanted to make sure you're quite confident on this one.

Reply:
SDS updated their equality check to also check the name field rather than object equality.  As of recent, this prop was renamed, so the one we're passing here isn't being used.  It shouldn't matter though since it no longer relies on object equality.

Reply:
Good to know, thanks!